azerothcore/azerothcore-wotlk
# Fix(DB/Quest): Wrong pre-requisite for quest 9706

## Changes Proposed:
-  Change the pre-requisite quest.

## Issues Addressed:

- Closes #8523
- Closes https://github.com/chromiecraft/chromiecraft/issues/2092

## SOURCE:

Wowhead

## Tests Performed:

- Tested in-game

## How to Test the Changes:


1. Get an Alliance character at lvl 15
2. ``.go gob 12183``
3. Observe the item Galaen's Journal can't not be looted and quest 9706 can't be started.
4. ``.q a 9578`` and complete with the npc besides the book.
5. Observe you can now take the quest 9706.
